content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: With Gateways
+description: Istio mesh spanning multiple Kubernetes clusters using Istio Gateway to reach remote pods.
+weight: 2
+keywords: [kubernetes,multicluster,federation,gateway]
+---
+
+Instructions for spanning an Istio mesh across multiple clusters when pods
+in each cluster can only talk to remote gateway IPs.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters with **1.9 or newer**.
+
+* The ability to deploy the [Istio control plane](/docs/setup/kubernetes/quick-start/)
+on **each** Kubernetes cluster.
+
+* The gateway IP (load balancer IP) in each cluster must be accessible
+from every other cluster.
+
+* **Root CA**: To enable mTLS communication across clusters, each cluster's
+  Citadel must be configured with an intermediate CA credentials generated
+  by a shared root CA.
+
+## Overview
+
+In this mode, each cluster has an **identical** Istio control plane
+installation. Cross-cluster communication occurs over Istio Gateways
+of the respective clusters.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-gateways.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using Istio Gateway to reach remote pods"
+    >}}
+
+## Deploy Istio control plane in each cluster
+
+1. Generate intermediate CA certs for each cluster's Citadel from your
+organization's root CA. The shared root CA enables mTLS communication
+across different clusters.
```

Comment:
WE can't because this is a much bigger thing (same as how to install kubernetes). CAs are hosted entities and different systems have different processes. Showcasing a dummy CA generation will make this a task instead of a realistic installation setup.